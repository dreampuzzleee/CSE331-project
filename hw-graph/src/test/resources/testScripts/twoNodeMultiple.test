# Tests graph creation/viewing with a simple graph ('A') with two nodes and multiple edges.

# Create a graph
CreateGraph A

# Add a pair of nodes
AddNode A n1
AddNode A n2

# Add an edge
AddEdge A n1 n2 e12a
AddEdge A n2 n1 e21a
AddEdge A n1 n1 e11
AddEdge A n2 n2 e22

# Print all nodes in the graph
ListNodes A

# Print all child nodes of n1, n2 with outgoing edge
ListChildren A n1
ListChildren A n2