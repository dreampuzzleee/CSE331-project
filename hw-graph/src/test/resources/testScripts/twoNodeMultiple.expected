# Tests graph creation/viewing with a simple graph ('A') with two nodes and multiple edges.

# Create a graph
created graph A

# Add a pair of nodes
added node n1 to A
added node n2 to A

# Add an edge
added edge e12a from n1 to n2 in A
added edge e21a from n2 to n1 in A
added edge e11 from n1 to n1 in A
added edge e22 from n2 to n2 in A

# Print all nodes in the graph
A contains: n1 n2

# Print all child nodes of n1, n2 with outgoing edge
the children of n1 in A are: n1(e11) n2(e12a)
the children of n2 in A are: n1(e21a) n2(e22)
